components/collapsible-panel/test/collapsible-panel.vdiff.js

```diff
@@ -0,0 +1,163 @@
+import '../../button/button-icon.js';
+import '../../link/link.js';
+import '../../status-indicator/status-indicator.js';
+import '../../typography/typography.js';
```

Comment:
The need for and application of the `d2l-typography` CSS class is a bit weird IMO. Wondering what that's doing...

Reply:
It's for lines 61 and 89 (`<p class="d2l-body-small">...</p>`).  I could instead just set some styles directly to try to get "close enough"

Reply:
I see. It's fine for the migration, but we probably want to avoid this kind of vdiff component cross-contamination in the future unless it's testing something very specific and common (e.g. different button components in overflow-group).

Reply:
Yeah agreed, my focus has been migrating exactly what's there but an audit wouldn't be a bad idea at some point